sequencing/ATACSeq/jobSubmission/0_setUp.sh

```diff
@@ -113,27 +114,35 @@ done
 
 ## Load required modules ##
 
-module load ${ACVERS}
-module load ${PVERS}
-module load ${RVERS}
+ml ${MCVERS}
+ml ${RVERS}
 
-## Check that the SAMstats environment exists and create it if not
+## Check that a conda environment exists and create it if not
 
 if [[ -d ${CONDA} ]]
 then 
 	echo "${PROJECT} conda environment exists"
 else
 	echo "${PROJECT} environment does not exist"
 	echo "Creating environment"
-  source ${CONDA_ENV}
+  source activate base #${CONDA_ENV}
   if [[ -f ${CONDA_FILE} ]]
   then
+    #conda create -n ${PROJECT}
 	  conda env create --name ${PROJECT} --file ${CONDA_FILE}
     conda deactivate
   else
     echo "File with packages to install in conda environment not found."
   fi
-  
+  if [[ -f ${PIP_FILE} ]]
+  then
+    source activate ${PROJECT}
+    pip install -r ${PIP_FILE}
+    conda deactivate
```

Comment:
Honestly with the amount of this pipeline that already relied on conda and pip, it makes sense to move away from the module based system anyways. It reduces some complexity. In my opinion this change can stay even if we stop having these issues on ISCA.

Reply:
Although, there is a problem here I think. Why are we still using the miniconda module command? Does this not give us the exact same problem? If the miniconda module doesn't load, we are in the exact same situation no? Why not just make miniconda a software requirement (which needs to be on `PATH`)?

Reply:
Interestingly, loading miniconda seems fine, but I will look for a way to avoid loading it

Reply:
There is three nodes that I have identified so far that will not load any modules at all. These have been given to support, but they are still active currently.

Reply:
Adding path to miniconda seems to be working fine, so I will change the scripts to avoid loading miniconda module and allow users to choose whether they use miniconda/anaconda/mamba

Reply:
Yeah that's a good idea.